### ISSUE TITLE
Markers event return null for event.square

### ISSUE THREAD

**Issue by djeetee** [shaack/cm-chessboard#113]
Hey shaack,

Setup: 
1) any position like below. Create a bunch of arrows from say h7-h6, g7-g6, etc. all good.
2) redraw the arrows (the idea is to remove the arrow when it is redrawn). for some random reason either the mousedown or mouseup event return null for event.square. 
3) on mousedown i save the square and then check its value on mouseup to make a match and then remove the arrow. This gets messed up when I get a null :(

It doesn't happen in any specific order or patter. 

<img width="348" alt="image" src="https://user-images.githubusercontent.com/612096/236650670-48accdf7-940d-4bfb-b5c0-c66834733b52.png">

<img width="354" alt="image" src="https://user-images.githubusercontent.com/612096/236650656-3d8d5670-0f99-482b-8c5e-5dddbe30183f.png">

<img width="465" alt="image" src="https://user-images.githubusercontent.com/612096/236650746-27120bc3-d6a0-4ec4-a478-9919cbbb7b3a.png">

<img width="465" alt="image" src="https://user-images.githubusercontent.com/612096/236650995-7c9195ee-5be1-4414-a3d2-52deca57528d.png">

Any reason why enableSquareSelect event would return null for the square?

Thank you,

**Comment by shaack** [comment]
How to you assign the event listener? Can you provide an online example-page?

**Comment by djeetee** [comment]
Hi Stefan, that css one-liner seems to have fixed it. So far I have not been able to reproduce. I'm keeping my logging in the code for the next few days to see if pops up again. 

That was excellent my friend. 

cm-chessboard rocks!

thank you.

**Comment by djeetee** [comment]
Hi Stefan,

the code is very simple and it’s not related to how I assign the event.

.square is randomly returned as null even before doing anything. My event handler is this:


  markersHandler(event) {
    if (event.type === SQUARE_SELECT_TYPE.primary) {
      return // no interested in left clicks
    }

    if (!event.square) {
      // FIXME: this is a hack. sometimes the event.square is undefined
      // this was fixed by adding: .arrows { pointer-events: none; } to the css
      console.log('no square. event is ', event)
      return
    }

    const { pendingMarker } = storeToRefs(useBoardStore()) // holds the starting square of the pending marker

    const boardId = event.chessboard.id

    switch (event.mouseEvent.type) {
      case 'mousedown':
        pendingMarker.value.square = event.square

        // set the default marker and arrow types
        pendingMarker.value.markerType = APP_MARKER_TYPE.primary // default color
        pendingMarker.value.arrowType = APP_ARROW_TYPE.primary // default type/color

        // adjust based on the modifier keys
        if (event.mouseEvent.ctrlKey) {
          // ctrl + right-click
          pendingMarker.value.markerType = APP_MARKER_TYPE.secondary
          pendingMarker.value.arrowType = APP_ARROW_TYPE.secondary
        } else if (event.mouseEvent.altKey) {
          // alt + right-click
          pendingMarker.value.markerType = APP_MARKER_TYPE.tertiary
          pendingMarker.value.arrowType = APP_ARROW_TYPE.tertiary
        }

        break

      case 'mouseup':
        if (pendingMarker.value.square === event.square) {
          this.addMarkerToCurrentMove(
            boardId,
            pendingMarker.value.markerType,
            event.square
          )
        } else {
          this.addArrowToCurrentMove(
            boardId,
            pendingMarker.value.arrowType,
            pendingMarker.value.square, // from square
            event.square // to square
          )
        }

        // either way, reset the pending marker
        pendingMarker.value = {}

        break
    }
  }


-George

**Comment by shaack** [comment]
I made so many changes to the cm-chessboard in the last days. Did you try out the current version 7.6.4? I also changed the handling of drawings from extensions. I think you also should update the css "chessboard.css" to the newest version.

**Comment by djeetee** [comment]
I'm on version 7.3.7 and you're already on 7.6.5 :)

I will update (hopefully nothing breaks!). Thank you.

**Comment by shaack** [comment]
Yes, wish you the best :) . I frequently update the cm-chessboard, when I need new features for my own chess-server. If something breaks, like in "cm-pgn" I will help.
